qcodes/instrument_drivers/Keysight/Infiniium.py

```diff
@@ -44,6 +44,12 @@ def __init__(self, name: str, instrument: Instrument, channel: int):
     def prepare_curvedata(self, acquire_mode: str = 'RTIMe') -> None:
         """
         Prepare the scope for returning curve data
+
+        Args:
+            acquire_mode: This argument sets the `acquire_mode` param on the
+            Infiniium oscilloscope instrument for the measurement. Default
+            value is `RTIMe` which means realtime mode and only one trigger
+            is used.
```

Comment:
Please indent all lines that are does not start with a parameter a further 4 spaces. 

```
            acquire_mode: This argument sets the `acquire_mode` param on the
                Infiniium oscilloscope instrument for the measurement. Default
                value is `RTIMe` which means realtime mode and only one trigger
                is used.
```

Reply:
Otherwise it ends up looking like this:

![Screenshot 2021-03-22 110235](https://user-images.githubusercontent.com/548266/111972834-2874c680-8afe-11eb-86c3-b78d49787abe.png)

Reply:
ok